civis/tests/test_io.py

```diff
@@ -398,6 +398,147 @@ def test_civis_file_to_table_table_doesnt_exist(
             **expected_kwargs
         )
 
+    @pytest.mark.civis_file_to_table
+    @mock.patch('civis.io._tables._process_cleaning_results')
+    @mock.patch('civis.io._tables._run_cleaning')
+    def test_civis_file_to_table_table_exists_all_sql_types_missing(
```

Comment:
I think we want `table_does_not_exist` for this test. We're testing that if all sql types are missing that we run cleaning? But if the table exists then we don't need to run cleaning so we probably want to change this to the table not existing or the `existing_table_rows = 'drop'`.

Reply:
Ah - I misread the condition statement. I was so confused.

Reply:
done